Tests/FoundationInternationalizationTests/CalendarRecurrenceRuleTests.swift

```diff
@@ -84,5 +136,5 @@ final class CalendarRecurrenceRuleTests: XCTestCase {
             Date(timeIntervalSince1970: 1730712000.0), // 2024-11-04T01:20:00-0800
         ]
         XCTAssertEqual(results, expectedResults)
-   }
```

Comment:
some unrelated whitespace changes here, these would have happened in one commit or another.